ding/envs/env_manager/base_env_manager.py

```diff
@@ -10,6 +10,8 @@
 import enum
 import time
 import traceback
+import gym
```

Comment:
unused gym?

Reply:
Used in observation/action/reward space attribute typing: `def action_space(self) -> gym.spaces.Space:`